quickwit-core/src/indexing/document_indexer.rs

```diff
@@ -28,9 +28,9 @@ use tokio::sync::mpsc::Sender;
 
 use crate::indexing::document_retriever::DocumentRetriever;
 use crate::indexing::split::Split;
-use crate::indexing::statistics::StatisticEvent;
 
 use super::IndexDataParams;
+use super::INDEXING_STATISTICS;
```

Comment:
Can we avoid the global here? It does not seem necessary and it prevents index_documents from being usable in a different context.